packages/extension-sample/src/hooks/useExtensionConfig.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {useApplicationExtension} from '@salesforce/pwa-kit-extension-sdk/react'
+import Extension from '../setup-app'
+
+export const useExtensionConfig = () => {
```

Comment:
NIT: If you are looking through the eyes of the extension developer, the config in that context is always the "extensions" config and not the base applications config. I'm wondering if we change this to `useConfig` at the extensions config is the first class citizen so to speak and the base application would be the applicationConfigurtation

I don't feel strongly about this tho.

Reply:
Good call, I guess I want to make sure those existing PWA Kit users to understand the distinction more easily, might be easier for those beta testers who upgrade and try extension development. I'm happy to change it to `useConfig` if you still think that's a better name.